src/media_analyze.py

```diff
@@ -19,19 +19,19 @@ def calculate_frames_moving_average(video_results, windowed_stats_sec):
     # frame, ts, video_result_frame_num_read_int
     video_results = pd.DataFrame(video_results.dropna(subset=["value_read"]))
 
-    video_results = video_results.dropna()
     if len(video_results) == 0:
         return pd.DataFrame()
     # only one testcase and one situation so no filter is needed.
     startframe = video_results.iloc[0]["value_read"]
     endframe = video_results.iloc[-1]["value_read"]
 
-    frame = startframe
+    frame = int(startframe)
```

Comment:
round() instead?

```
>>> int(2.8)
2
>>> round(2.8)
3
```

Reply:
It will be either 0 or 0.5 so round have limited value (unless we have a randomized rounding).
The frame matching is difficult in float mode and the exact start is not fundamentally important.

Reply:
We can change the behavior but I do not think it matters (https://docs.python.org/3/library/decimal.html#rounding-modes:~:text=in%20version%203.8.3.-,Rounding,-modes%C2%B6)